vault_core/src/vault_operator_delegation.rs

```diff
@@ -57,10 +58,15 @@ impl VaultOperatorDelegation {
         self.index.into()
     }
 
-    pub fn is_update_needed(&self, slot: u64, epoch_length: u64) -> bool {
+    pub fn is_updated(&self, slot: u64, epoch_length: u64) -> Result<(), VaultError> {
```

Comment:
nit: Change to `check_is_already_updated` to match naming convention of error throwing checks

Reply:
Thanks!